src/test/java/com/wikia/webdriver/common/contentpatterns/PageContent.java

```diff
@@ -4,143 +4,126 @@
+  public static final String FORUM_DESCRIPTION_PREFIX =
       "Duis quam ante, fringilla at cursus tristique ";
-  public static final String
-      FORUM_DESCRIPTION_EDIT_PREFIX =
+  public static final String FORUM_DESCRIPTION_EDIT_PREFIX =
       "Duis quam ante, fringilla at cursus tristique edit ";
   public static final String FORUM_MESSAGE = "QAforumMessage";
   public static final String FORUM_BOARD = "QABoardForMoveThreadTest";
   public static final String CLOSE_REASON = "QA reason";
 
-  //Messages
+  // Messages
   public static final String LOGIN_REQUIRED = "Login required";
   public static final String NOT_LOGGED_IN_MESSAGE = "Not logged in";
-  public static final String
-      NEW_PASSWORD_SENT_MESSAGE =
+  public static final String NEW_PASSWORD_SENT_MESSAGE =
       "We've sent a new password to the email address for %userName%.";
-  public static final String
-      PHALANX_BLOCK_TITLE_MESSAGE =
+  public static final String PHALANX_BLOCK_TITLE_MESSAGE =
       "Sorry, the page title was rejected by our spam filter. Please use a different title.";
-  public static final String
-      SIGN_UP_TOO_YOUNG_MESSAGE =
+  public static final String SIGN_UP_TOO_YOUNG_MESSAGE =
       "Sorry, we're not able to register your account at this time.";
-  public static final String
-      SIGN_UP_INVALID_CAPTCHA_MESSAGE =
+  public static final String SIGN_UP_INVALID_CAPTCHA_MESSAGE =
       "The word you entered didn't match the word in the box, try again!";
-  public static final String
-      SIGN_UP_USER_EXISTS_MESSAGE =
+  public static final String SIGN_UP_USER_EXISTS_MESSAGE =
       "Someone already has this username. Try a different one!";
 
-  //wikiText
+  // wikiText
   public static final String WIKI_TEXT_PHOTO = "[[File:%photoName%|thumb|%s]]";
 
-  //SignUp correct
+  // SignUp correct
   public static final String WIKI_SIGN_UP_BIRTHDAY = "11";
   public static final String WIKI_SIGN_UP_BIRTHMONTH = "11";
   public static final String WIKI_SIGN_UP_BIRTHYEAR = "1954";
   public static final int MIN_AGE = 12;
 
-  //Hero Module edit field default message
-  public static final String
-      WIKIA_HERO_PROMOTE_MESSAGE =
+  // Hero Module edit field default message
+  public static final String WIKIA_HERO_PROMOTE_MESSAGE =
       "Add a summary that will promote your wikia in searches and on hubs.";
 
-  //Portable Infoboxes content
+  // Portable Infoboxes content
   public static final String PORTABLE_INFOBOX02 = "Infobox4Automation01";
   public static final String PORTABLE_INFOBOX01 = "Infobox4Automation02";
   public static final String PORTABLE_INFOBOX_EMPTY_TAGS = "Infobox4Automation09";
   public static final String PI_TEMPLATE_WEBSITE_SIMPLE = "Template:Infobox_Website_Simple";
   public static final String FILE_IMAGE_NAME = "File:WallPaperHD_138.jpg";
 
-  //Customized toolbar
+  // Customized toolbar
   public static final String FOLLOW = "follow";
 
-  private PageContent() {
-  }
+  private PageContent() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this closing curly brace to the next line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ARightCurlyBraceStartLineCheck)